src/api/server.rs

```diff
@@ -38,7 +38,7 @@ impl Server {
         running: Arc<AtomicBool>,
         handle: &Handle,
     ) -> crate::Result<Self> {
-        let routes = make_routes(config.api.playground, watch_rx, running);
+        let routes = make_routes(config.api.graphql, config.api.playground, watch_rx, running);
```

Comment:
Would it make sense to pass `&config.api` instead of adding another parameter to `make_routes`?

Reply:
Ya good idea 👍